lib/dpul_collections/solr.ex

```diff
@@ -14,12 +15,53 @@ defmodule DpulCollections.Solr do
     response.body["response"]["numFound"]
   end
 
+  def project_summary(label, index \\ Index.read_index()) do
+    params =
+      SearchState.from_params(%{
+        "filter" => %{"project" => label},
+        "per_page" => "0"
+      })
+      |> Map.put(
+        :extra_params,
+        facet: true,
+        "facet.field": "language_txt_sort",
+        "facet.field": "geographic_origin_txt_sort",
+        "facet.field": "categories_txt_sort",
+        "facet.field": "genre_txt_sort",
+        "facet.sort": "count",
+        "facet.mincount": 1,
+        "facet.limit": -1
+      )
+
+    response = raw_query(params, index)
+    # Returns something like
+    # %{ "response" => solr_response, "facets" => %{"solr_field" => [{"Value", count}, {"Value 2", count}]}}
+    response["response"] |> Map.merge(%{"facets" => extract_facets(response["facet_counts"])})
+  end
+
+  defp extract_facets(%{"facet_fields" => facet_fields}) do
+    # facet_fields looks like %{"field" => ["Label", count]}
+    facet_fields
+    |> Enum.map(fn {field, field_values} ->
+      {
+        field,
+        # Split into pairs
+        Enum.chunk_every(field_values, 2)
+        # Convert sub-lists to tuples
+        |> Enum.map(&List.to_tuple/1)
+      }
+    end)
+    |> Map.new()
+  end
+
   @query_field_list [
     "id",
     "title_ss",
     "display_date_s",
     "file_count_i",
     "detectlang_ss",
+    "authoritative_slug_s",
```

Comment:
This is a little confusing, if there's an authoritative slug then what does that mean about the other slug? Why can't we use the other slug field for collection slugs, given that we have the resource type?

Reply:
Oh yes I can probably refactor this out now. The reason I had it was because I was using Item structs in the search results view and things were gettin' weird for some reason, now that I use the collection structs I think we can do better. Thank you

Reply:
OH, wait, no, the reason is because Solr's generating `slug_s` and it overrode whatever I put there.